WooCommerce/WooCommerceTests/System/WooAnalyticsTests.swift

```diff
@@ -90,6 +90,36 @@ class WooAnalyticsTests: XCTestCase {
         }
     }
 
+    /// Verifies an event with an error and properties is received by the AnalyticsProvider
+    ///
+    func testEventsWithPropertiesAndErrorReceived() {
```

Comment:
nit: we use snake case for test case names now based on [this documentation](https://github.com/woocommerce/woocommerce-ios/blob/c5d83703d6c0565fc34bf410b6b27483f2318df2/docs/naming-conventions.md#test-methods) 😄 
```suggestion
    func test_events_with_properties_and_error_include_combined_properties() {
```

Reply:
Thanks. Again I was going for consistency, and again I think that was the wrong call.